gbasis/integrals/point_charge.py

```diff
@@ -271,6 +271,12 @@ def point_charge_integral(
 ):
     r"""Return the point-charge interaction integrals of basis set in the given coordinate systems.
 
+    .. math::
+
+        \int \phi_a(\mathbf{r}) \frac{1}{|\mathbf{r} - \mathbf{R}_C|} \phi_b(\mathbf{r}) d\mathbf{r}
```

Comment:
@gabrielasd, I believe there should be a sum in this equation based on the docstring and implementation.

Reply:
Oh, I understood the problem was a missing summation over the point charges.

@PaulWAyers I'm actually confused about the implementation of the formula for this function, `point_charge_integral`, Eq 47 page 21 in notes.pdf, in contrast with the one for `nuclear_electron_attraction_integral`, Eq. 47 page 21. To me the only difference in these equations is the value of the charge. But the implementations of both are totally different.

This function, `point_charge_integral`, like Farnaz points out, is returning a matrix of shape: number of basis functions by number of point charges, which is not reflected by the added formula. On the other hand `nuclear_electron_attraction_integral` returns a square matrix of the dimensions of the basis functions.
![point_chargeEq47](https://github.com/theochem/gbasis/assets/26312264/994d6c1d-4b3a-46a8-b312-1f0505d972e6)
![nuc_electron_attractionEq48](https://github.com/theochem/gbasis/assets/26312264/afb70671-7f19-4bb7-9d30-55e10ab4b280)

Reply:
I ran the examples for both functions, and compared the shape of the returned variables; it seems to me that the documented return of `point_charge_integral` is missing one dimension:
https://github.com/theochem/gbasis/blob/9f6afe7ffc56b9a4e7f8a4a74d0fa5f01197ebca/gbasis/integrals/point_charge.py#L291
and should be `np.ndarray(K, K, N)` where K is the number of basis functions.

I'll add a comment to the docstring of Eq. 47 clarifying that what is returned are these (one-electron) integral matrix terms per every point charge, given that I'm not sure how to express it in the equation. For Eq. 47 I remember adding a sum over the charges in a previous commit.

Reply:
Hi @gabrielasd, I was checking your PR and specifically this part. I agree with you, I run the code and the `point_charge` integral function returns a matrix of elements being integrals of pairs of basis functions for each of the charges passed. Actually I was less sure about this line `N` is the number of coordinates at which the contractions are evaluated.` This would be charge points not coordinates right? Let me know what you think.

Reply:
Hi @leila-pujal, yes I agree with you. 
It does seem a bit ambiguous since in this case the number of coordinates should match the number of charge points.